site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,390 @@
+
+### Add support for SDL controller accelerometer/gyro events ([#5851](https://github.com/citra-emu/citra/pull/5851)) by [flibitijibibo](https://github.com/flibitijibibo)
+
+Following the release of SDL 2.0.14 came support for motion controls, something which had been anticipated for many years in SDL. In Citra, this can be used as an alternative to mouse controls or CemuhookUDP. This is a big benefit to many users, as setting up Cemuhook can be tedious and outright confusing at times. 
+Setting up SDL motion controls is as easy as pressing a button! (No, seriously. That is all you have to do.)
+
+{{< figure src="motion.png"
+    title="Motion controls at the press of a button!" >}}
+
+# Emulation Accuracy
+
+### Implement APT command 0x0103 ([#5478](https://github.com/citra-emu/citra/pull/5478)) by [xperia64](https://github.com/xperia64)
+
+This PR allows for controls introduced by the New Nintendo 3DS to be utilized in Citra whilst **New 3DS mode** is enabled. The New Nintendo 3DS came with the addition of the C-Stick, and the ZL and ZR trigger buttons. 
+These can now be used in games such as Super Smash Bros. for Nintendo 3DS to give you more control over how you play your game!
+
+### Initial implementation of partial_embedded_buffer_dirty handling ([#5548](https://github.com/citra-emu/citra/pull/5548)) by [xperia64](https://github.com/xperia64)
+
+For a while, the Mii voices in Tomodachi Life would continuously cut off before they would finish speaking, which could make it hard to understand what they are saying to the player. By implementing a partial embedded buffer, a buffer being an area of memory which stores data on a temporary basis, you can hear the sweet voices of your favorite Miis again!
+
+This also fixes an issue where FMVs in Detective Pikachu would run faster than what the frame rate allowed for.
+
+### service/apt: Implement Deliver Args ([#5611](https://github.com/citra-emu/citra/pull/5611)) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+This partially fixes an issue where some games would just show black and white screens when launched. By implementing Deliver Args, a way to pass arguments from the caller to the callee, as well as `PrepareToDoApplicationJump` and `JumpToApplication` APT services. This allows the game to crash with a fatal error instead of just hanging on a black screen. 
+
+This is a step in the right direction for getting these games, such as Sega 3D Classics Collection, to boot correctly on Citra in the future.
+
+### gl_rasterizer_cache: Remove all fully invalid surfaces from the cache ([#5710](https://github.com/citra-emu/citra/pull/5710)) by [BreadFish64](https://github.com/BreadFish64)
+
+The dead weight of fully invalid surfaces being held in the rasterizer cache impacted many cache management functions and texture lookup times, which in turn impacted performance on some games, such as Pilotwings.
+
```

Comment:
Nag @BreadFish64 to give you an estimated % bump.

Reply:
I've got a graph in that PR. I didn't do hard measurements on any other games, it's highly dependent on how the games behave when allocating framebuffers.